src/components/index.ts

```diff
@@ -9,6 +9,7 @@ export {
   default as AppProvider,
```

Comment:
Need to use these in `src/hooks` and we can't dive into component folders from outside components